addons/settings/fnc_gui_settingTime.sqf

```diff
@@ -0,0 +1,114 @@
+#include "script_component.hpp"
+
+params ["_controlsGroup", "_setting", "_source", "_currentValue", "_settingData"];
+_settingData params ["_min", "_max"];
+
+private _ctrlSlider = _controlsGroup controlsGroupCtrl IDC_SETTING_TIME_SLIDER;
+_ctrlSlider setVariable [QGVAR(params), [_setting, _source]];
+
+_ctrlSlider sliderSetRange [_min, _max];
+_ctrlSlider sliderSetPosition _currentValue;
+private _range = _max - _min;
+_ctrlSlider sliderSetSpeed [0.05 * _range, 0.1 * _range];
+
+_ctrlSlider ctrlAddEventHandler ["SliderPosChanged", {
+    params ["_ctrlSlider", "_value"];
+    (_ctrlSlider getVariable QGVAR(params)) params ["_setting", "_source"];
+    _value = round _value;
+
+    private _controlsGroup = ctrlParentControlsGroup _ctrlSlider;
+
+    {
+        _x params ["_idc", "_value"];
+
+        private _ctrlEdit = _controlsGroup controlsGroupCtrl _idc;
+        private _editText = if (_value < 10) then {format ["0%1", _value]} else {str _value};
```

Comment:
Replace with `CBA_fnc_formatNumber`?

Reply:
Could do, but this works too. Only difference I believe would be rounding instead of truncating.